k8s/openebs-operator.yaml

```diff
@@ -363,7 +363,7 @@ spec:
       hostNetwork: true
       containers:
       - name: node-disk-manager
-        image: quay.io/openebs/node-disk-manager-amd64:v0.3.5
```

Comment:
@kmova and @akhilerm changed it to v0.3.5 in just recent checkin

Reply:
@vishnuitta reverted the change because the same file is used to deploy openebs in `openebs/maya` travis since there is no support for blockdevice in maya it reverted back to `v0.3.5`. If PR: https://github.com/openebs/maya/pull/1227 needs to merge to support pool creation on blockdevice in travis these change is required.